tsconfig.json

```diff
@@ -47,6 +48,10 @@
       }
     ]
   },
+  "include": [
+    ".eslintrc.js",
```

Comment:
this is urgly, it's so that the parser can recognize it. 
I can change this to a json file, and it'll pick up implicitly. To be consistent I'd change the prettier config as well.

Reply:
Love this! Also like that you moved this to json below, that was probably gonna be my main point of feedback haha